code/modules/antagonists/bloodsucker/bloodsucker_powers.dm

```diff
@@ -197,10 +197,6 @@
 	button.screen_loc = DEFAULT_BLOODSPELLS
 	button.moved = DEFAULT_BLOODSPELLS
 	button.ordered = FALSE
-/datum/action/bloodsucker/passive/Destroy()
-	if(owner)
-		Remove(owner)
-	target = null
```

Comment:
part of this is done in the parent Destroy() proc, I'm not totally sure why this was overriden, or what behavior it is trying to avoid